tests/Unit/SDK/Resource/Detectors/ConstantTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Unit\SDK\Resource\Detectors;
+
+use OpenTelemetry\SDK\Common\Attribute\Attributes;
+use OpenTelemetry\SDK\Resource\Detectors;
+use OpenTelemetry\SDK\Resource\ResourceInfo;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers OpenTelemetry\SDK\Resource\Detectors\Constant
+ */
+class ConstantTest extends TestCase
+{
+    public function test_constant_get_resource_with_empty_resource(): void
+    {
+        $resouceDetector = new Detectors\Constant(ResourceInfo::create(new Attributes()));
+        $resource = $resouceDetector->getResource();
+
+        $this->assertNull($resource->getSchemaUrl());
```

Comment:
See above regarding schema url, but again this doesn't need to be addressed in this PR, as I'm not sure about the purpose of this detector in the first place.